src/modelfree/ppo_baseline.py

```diff
@@ -22,6 +22,9 @@
 
 from aprl.envs.multi_agent import CurryEnv, FlattenSingletonEnv
 from modelfree.gym_compete_conversion import GymCompeteToOurs, load_zoo_agent
+from modelfree.reward_shaping import (LinearAnnealer, RewardShapingEnv, Scheduler,
```

Comment:
Classes should go first, and be in alphabetical order, then functions. isort will do this for you automatically, please run ci/code_checks.sh